src/components/FileUploader/FileUploader.js

```diff
@@ -1,27 +1,24 @@
 import React from 'react';
 import PropTypes from 'prop-types';
-import { Box, makeStyles, Typography, Tooltip } from '@material-ui/core';
+import { Box, makeStyles } from '@material-ui/core';
 import { DropzoneDialog } from 'material-ui-dropzone';
-import { useTranslation } from 'react-i18next';
 import Button from '../Button';
 
 export const useStyles = makeStyles(theme => ({
     error: { color: theme.palette.error.main },
-    button: { marginRight: theme.spacing(2) },
+    button: { marginRight: theme.spacing(0) },
 }));
 
 const FileUploader = ({
     className,
     style,
     onSave,
-    error,
     acceptedFiles,
+    caption,
 }) => {
     const classes = useStyles();
     const [filename, setFilename] = React.useState('');
```

Comment:
We no longer need to set the filename internally within the component. This state will no longer have any UI element be dependent on it.